docs/deploying-java-app-with-database.md

```diff
@@ -0,0 +1,290 @@
+
+### Binding the database and the application
+
+Now, the only thing that remains is to connect the DB and the application. We will use odo to create a link to the Dev4Devs PostgreSQL Database Operator in order to access the database connection information.
+
+1.  Display the services available to odo: - You will see an entry for the PostgreSQL Database Operator displayed:
+    
+    ``` sh
+    $ odo catalog list services
+    Operators available in the cluster
+    NAME                                             CRDs
+    postgresql-operator.v0.1.1                       Backup, Database
+    ```
+
+2.  List the service associated with the database created via the PostgreSQL Operator:
+    
+    ``` sh
+    $ odo service list
+    NAME                        AGE
+    Database/sampledatabase     6m31s
+    ```
+
+3.  Create a Service Binding Request between the application and the database using the Service Binding Operator service created in the previous step `odo link` command:
+    
+    ``` sh
+    $ odo link Database/sampledatabase
+    ```
+
+4.  Push this link to the cluster:
+    
+    ``` sh
+    $ odo push
+    ```
+    
+    After the link has been created and pushed a secret will have been created containing the database connection data that the application requires.
+    
+    You can inspect the new intermediate secret via the dashboard console in the 'my-postgresql-operator-dev4devs-com' namespace by navigating to Secrets and clicking on the secret named `mysboproj-database-sampledatabase` Notice it contains 4 pieces of data all related to the connection information for your PostgreSQL database instance.
+
+5.  Push the newly created link. This will terminate the existing application pod and start a new application pod.
+    
+    ``` sh
+    odo push
+    ```
+
+6.  Once the new pod has initialized you can see the secret database connection data as it is injected into the pod environment by executing the following:
+    
+    ``` sh
+    $ odo exec -- bash -c 'export | grep DATABASE'
+    declare -x DATABASE_CLUSTERIP="10.106.182.173"
+    declare -x DATABASE_DB_NAME="sampledb"
+    declare -x DATABASE_DB_PASSWORD="samplepwd"
+    declare -x DATABASE_DB_USER="sampleuser"
+    ```
+    
+    Once the new version is up (there will be a slight delay until application is available), navigate to the CreatePerson.xhtml using the URL created in a previous step. Enter requested data and click the **Save** button.
+    
+    Notice you are re-directed to the `PersonList.xhtml` page, where your data is displayed having been input to the postgreSQL database and retrieved for display purposes.
+    
+    You may inspect the database instance itself and query the table to see the data in place by using the postgreSQL command line tool, psql.
+
+7.  Navigate to the pod containing your db from the Kubernetes Dashboard
+
+8.  Click on the terminal tab.
+
+9.  At the terminal prompt access psql for your database
```

Comment:
"9.  At the terminal prompt, access psql for your database."
Missing comma before "access" and a full stop or color at the end.